src/cloudformation_cli_python_lib/resource.py

```diff
@@ -211,9 +212,40 @@ def __call__(
             ProviderLogHandler.setup(event_data)
             parsed = self._parse_request(event_data)
             caller_sess, platform_sess, request, action, callback, event = parsed
+            # Acknowledge the task for first time invocation
+            if not event.requestContext:
```

Comment:
i think java also checks the invocation count if requestContext happens to be set: https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/d2cb70716ea42d87549b61d09a82d7abcb5c16e7/src/main/java/software/amazon/cloudformation/LambdaWrapper.java#L322